notebooks/Imputation_best_practices/Imputation_best_practices.ipynb

```diff
@@ -0,0 +1,4557 @@
+{
```

Comment:
<p>Is this essentially a list of 1000 random numbers? If yes, then could we directly generate it as <code>random_dataset = pd.DataFrame({'number': np.random.rand(n)})</code>  instead of adding a separate csv to the repo?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/data-science-workflows/pull/113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Yes, this can be done!</p>